### DISCUSSION TITLE
Pull requests

### DISCUSSION THREAD

**Discussion by ptav** [coddingtonbear/django-mailbox#261] [General]
Hi, I've had to switch across to @skar395 fork due to compatibility issues and have now been using that for a number of months in our internal CRM. All seems good so can probably merge it into master (we poll emails every few minutes, probably a few hundred a day). 

Can't comment on Django 4 compatibility as we're still on v2

@coddingtonbear happy to help out on that if useful

**Comment by coddingtonbear** [comment]
Absolutely!  If you happen to have tested any of the branches of code posted here in production (or, really, in any environment), that's exactly the sort of thing that I need somebody to do before a pull request can be merged.

Do you happen to know what the overlap is between the @skar935 fork and the posted PRs?

**Reply by Pietro395** [reply]
Hi @coddingtonbear , What do you mean?
However, we are also using our fork in production with the support to Office365 without problems

**Reply by coddingtonbear** [reply]
Hey there @skar395 --

What I was talking about with @ptav is that one of the main problems we have on this project is getting insight into when particular pull requests have been tested by anybody, and to what level they were tested (i.e. actively running it in production vs just having opened the django admin with the PR merged, or some level between those ends of the spectrum).  Manually testing of a particular PR is one of the more time-consuming things that needs to happen before a pull request can be merged, and, unfortunately, as much as I might want to, I'm not going to have time to personally test anything until I retire...somewhere after 2055.

What I was asking @ptav to do, if he could, is note on pull requests he has tested in production so we could get an indication of what pull requests are known to actually work.  That'll get those pull requests one step closer to merging.

**Reply by ptav** [reply]
Will do!
